solutions/fully-configurable/variables.tf

```diff
@@ -114,21 +114,35 @@ variable "existing_en_instance_crn" {
 # KMS
 ########################################################################################################################
 
+variable "kms_encryption_enabled" {
+  type        = bool
+  description = "Set to true to enable KMS encryption on Event Notifications instance and Cloud Object Storage bucket. When set to true 'kms_endpoint_url' and one of 'existing_kms_instance_crn' or 'existing_kms_root_key_crn' must be set."
+  default     = true
```

Comment:
This should not default to true in fully configurable because the user is then required to pass either a KMS instance CRN, or a KMS key. We want fully configurable to be like a one click deploy. This is consistent with all our other fully configurable DAs